.changelog/3939.bugfix.md

```diff
@@ -0,0 +1,13 @@
+Update emitted events when disbursing rewards
```

Comment:
This may break integrations for custody providers which are parsing events for rewards. Should reclassify as breaking.

Reply:
Fixed. But do we want this change right? An alternative is also keeping it as it is and just having it documented

Reply:
Yeah I think it makes tracking rewards more clear. We should just give a heads up when we release this.